staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -649,6 +650,7 @@ func TestCacherNoLeakWithMultipleWatchers(t *testing.T) {
 
 	// run the collision test for 3 seconds to let ~2 buckets expire
 	stopCh := make(chan struct{})
+	errc := make(chan error, 1)
```

Comment:
You can just use a pointer to an error. The channel isn't necessary.

Reply:
Thanks a lot for your feedback! Replaced this and the other three superfluous error channels in https://github.com/kubernetes/kubernetes/pull/103023/commits/cd773a2e941450ba44f6f4edd6b0f8ea3c01fcae.